sample/github_hmac/Dockerfile

```diff
@@ -2,13 +2,13 @@ FROM golang:1.7.5-alpine
 
 MAINTAINER alexellis2@gmail.com
 ENTRYPOINT []
-#RUN mkdir -p /go/src/github.com/alexellis/faas/sample-functions/handler
-WORKDIR /go/src/github.com/alexellis/faas/sample-functions/handler
-COPY . /go/src/github.com/alexellis/faas/sample-functions/handler
+#RUN mkdir -p /go/src/github.com/openfaas/faas/sample-functions/handler
```

Comment:
Is there an additional change here?

Reply:
do you mean the commented out line? no that was already there